Integrate gutenberg-mobile release 1.47.0

antonis opened the pull request:
## Description
This PR incorporates the 1.47.0 release of gutenberg-mobile.  
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3155
Release Submission Checklist
- [x] I have considered if this change warrants user-facing release notes and have added them to \`RELEASE-NOTES.txt\` if necessary.

ceyhun committed:
Hide audio block on free sites

fluiddot committed:
Release script: Update gutenberg-mobile ref

ceyhun committed:
Update gutenberg-mobile ref

jd-alexander committed:
Updated gutenberg-mobile ref.

jd-alexander committed:
Merge pull request #15865 from wordpress-mobile/feature/show-hide-audio-block

[Gutenberg] Hide audio block on free sites

fluiddot committed:
Update gutenberg-mobile ref

fluiddot committed:
Update release notes

fluiddot committed:
Revert cocoapods version change in Podfile.lock

fluiddot committed:
Update gutenberg-mobile ref

fluiddot committed:
Revert "Update release notes"

This reverts commit c855d99753acaa2f0e7d16a08d4ce960c616f449.

antonis committed:
Release script: Update gutenberg-mobile ref

dcalhoun committed:
Update gutenberg ref

dcalhoun committed:
Install pods

dcalhoun committed:
Update release notes

dcalhoun reviewed (commented):
This is looking good to me so far. Hopefully the testing build finishes soon.

Based on https://github.com/wordpress-mobile/WordPress-iOS/pull/15752#pullrequestreview-584197256, we'll need to swap from a git commit to a tag after we merge wordpress-mobile/gutenberg-mobile#3155.

fluiddot committed:
Update gutenberg-mobile ref

fluiddot committed:
Update gutenberg-mobile ref

fluiddot committed:
Merge branch 'gutenberg/integrate_release_1.47.0' into gutenberg/integrate_release_1.46.1_to_1.47.0

# Conflicts:
#	Podfile
#	Podfile.lock

fluiddot committed:
Update Pod files due to bad conflict resolution

dcalhoun committed:
Update gutenberg ref

dcalhoun committed:
Install pods

dcalhoun committed:
Merge branch 'gutenberg/integrate_release_1.47.0' of github.com:wordpress-mobile/WordPress-iOS into gutenberg/integrate_release_1.46.1_to_1.47.0

fluiddot committed:
Merge pull request #15905 from wordpress-mobile/gutenberg/integrate_release_1.46.1_to_1.47.0

Integrate gutenberg-mobile release 1.46.1 to 1.47.0

antonis committed:
Merge branch 'develop' into gutenberg/integrate_release_1.47.0

antonis committed:
Updated Podfile.lock

dcalhoun commented on Podfile:
```diff
@@ -153,7 +153,7 @@ target 'WordPress' do
     ## =====================
     ##
-    gutenberg :tag => 'v1.47.0-alpha2'
+    gutenberg :commit => '7f75b334f402ff36ac154274b728c25533dcb3a4'
```
Prior to merging, we'll need to create a `v1.47.0` in `gutenberg-mobile`, reference that here, and install the pods.

dcalhoun reviewed (approved):
LGTM. We just need to change the Podfile ref from commit to tag before merging.

dcalhoun committed:
Update gutenberg-mobile ref to v1.47.0 tag

dcalhoun committed:
Merge branch 'develop' of github.com:wordpress-mobile/WordPress-iOS into gutenberg/integrate_release_1.47.0

fluiddot commented on RELEASE-NOTES.txt:
```diff
@@ -2,6 +2,10 @@
 * [**] Prevent deleting published homepages which would have the effect of breaking a site. [#15797]
 * [**] Prevent converting published homepage to a draft in the page list and settings which would have the effect of breaking a site. [#15797]
+* [*] Block Editor: Fixed block mover title wording for better clarity from 'Move block position' to 'Change block position'. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3049]
+* [**] Block Editor: Make inserter long-press options "add to beginning" and "add to end" always available. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3074]
```
👋 This change was included in the `1.46.1` beta fix so it should better go under `16.7` section.

fluiddot commented on RELEASE-NOTES.txt:
```diff
@@ -2,6 +2,10 @@
+* [*] Block Editor: Fixed block mover title wording for better clarity from 'Move block position' to 'Change block position'. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3049]
+* [**] Block Editor: Make inserter long-press options "add to beginning" and "add to end" always available. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3074]
+* [**] Block Editor: Add support for setting Cover block focal point. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3028]
+* [*] Block Editor: Fix crash when Column block width attribute was empty. [https://github.com/WordPress/gutenberg/pull/29015]
```
The same here, this was included in `1.46.1` beta fix so better to have it under `16.7` section.

fluiddot reviewed (commented):
👋 I've found some discrepancies in the release notes related to the changes included the beta fixes.

dcalhoun committed:
Fix release notes accuracy

Relocate changes that were made in a beta fix and reside in the previous
16.7 release.

dcalhoun committed:
Merge branch 'develop' of github.com:wordpress-mobile/WordPress-iOS into gutenberg/integrate_release_1.47.0
